v2/fleet/sumologic-journald.service

```diff
@@ -0,0 +1,30 @@
+[Unit]
+Description=Sumologic Collector
+Requires=docker.service
+Before=journald-sumo-pipe.service
+After=docker.service bootstrap.service
+
+[Service]
+EnvironmentFile=/etc/environment
+User=core
+Restart=always
+TimeoutStartSec=0
+ExecStartPre=/usr/bin/docker pull behance/docker-sumologic
+ExecStartPre=-/usr/bin/docker kill sumologic-journald
+ExecStartPre=-/usr/bin/docker rm -f sumologic-journald
+ExecStart=/usr/bin/bash -c \
+"if [[ -f /etc/profile.d/etcdctl.sh ]]; then source /etc/profile.d/etcdctl.sh;fi && \
+sudo /usr/bin/docker run --name sumologic-journald \
+-p 514:514 \
+-p 514:514/udp \
+-e PATH_EXPRESSION=/syslog/syslog \
+-e SUMO_NAME=${NODE_PRODUCT}-${NODE_TIER}-cluster \
+-e SUMO_CATEGORY=be/${NODE_PRODUCT}/${NODE_TIER}/journald-logs \
```

Comment:
Will we be able to filter journald logs for control, proxy and worker ASGs with this sourceCategory?